src/packages/translations/translations/en.json

```
@@ -156,11 +156,22 @@
     },
     "bridgeGasFee": {
       "label": "Bridge gas fee",
-      "tooltip": "TRANSLATION NEEDED"
+      "tooltip": "Used to cover the gas cost for sending your transfer on the destination chain"
     },
     "connectWalletButton": {
       "label": "Connect wallet"
     },
+    "errors": {
+      "dailyTransactionLimitExceeded": {
+        "message": "You cannot bridge more than {{readableAmountTokens}} ({{readableAmountUsd}}) on the destination chain due to the 24-hour limit. This limit will be reset on {{ date, dd MMM yyyy h:mm a }}",
+        "submitButton": "Amount entered higher than the daily limit"
+      },
+      "singleTransactionLimitExceeded": {
+        "message": "You cannot bridge more than {{readableAmountTokens}} ({{readableAmountUsd}}) on the destination chain in a single transaction",
+        "submitButton": "Amount entered higher than the limit"
+      },
+      "doesNotHaveEnoughNativeFunds": "Not enough {{token}} for this transaction"
```

I know you used the copy that's in Figma, but I think we should stay consistent with another similar use case we have in the dApp: the supply/repay forms.
So I'd go with the message:
```suggestion
      "doesNotHaveEnoughNativeFunds": "Insufficient {{tokenSymbol}} balance"
```